smithy-aws-protocol-tests/model/restJson1/http-payload.smithy

```diff
@@ -268,3 +269,61 @@ structure NestedPayload {
     greeting: String,
     name: String,
 }
+
+
+/// This examples uses a `@mediaType` trait on the payload with a string payload
+@http(uri: "/HttpStringPayloadHttpStringPayloadWithMediaTypeTrait", method: "POST")
+operation HttpStringPayloadHttpStringPayloadWithMediaTypeTrait {
+    input: HttpStringPayloadHttpStringPayloadWithMediaTypeTraitInputOutput ,
+    output: HttpStringPayloadHttpStringPayloadWithMediaTypeTraitInputOutput
+}
+
+apply HttpStringPayloadHttpStringPayloadWithMediaTypeTrait @httpRequestTests([
+    {
+        id: "RestJsonHttpStringPayloadWithMediaType",
+        documentation: "Serializes a string in the payload with a media type",
+        protocol: restJson1,
+        method: "POST",
+        uri: "/HttpStringPayloadHttpStringPayloadWithMediaTypeTrait",
+        body: "stringy String string",
+        bodyMediaType: "text/plain",
+        headers: {
+            "X-Foo": "Foo",
+            "Content-Type": "image/jpeg"
```

Comment:
so is bodyMediaType=text/plain because it is a `string` type? And Content-Type=image/jpeg because of the `@mediaType` trait?

I think the intent may be clearer by using below?
```
@mediaType("image/jpeg")
string JpegString
```
and
```
    @httpPayload
    jpegString: JpegString
```
and maybe 
```
        params: {
            foo: "Foo",
            jpegString: "This is definitely a jpeg"
        }
```